lib/autoload/trackable_job.rb

```diff
@@ -20,6 +20,8 @@ class Job < ApplicationRecord
 
     validates :redirect_to, absence: true, if: :submitted?
     validates :error, absence: true, unless: :errored?
+    validates :status, presence: true
```

Comment:
Should validate integer.

Reply:
schema_validations didn't validate numericality because this is represented as an enum in the model.